src/components/pages/Mentor/MentorHome.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { fetchMentors } from '../../../state/actions';
+// import mentorList from './MentorList'
+
+const MentorHome = props => {
+  const { profile } = props;
+
+  useEffect(() => {
+    props.fetchMentors(1); // change this later with login
+  }, []);
+  console.log(profile);
+
+  return (
+    <div>
+      <h1 style={{ textAlign: 'center' }} className="page-title">
+        Mentor Home Page
+      </h1>
+    </div>
+  );
+};
+
+const mapStateToProps = state => {
```

Comment:
Not something you need to change but, did you know that you do not need a return statement if you are just returning an object literal. There is just 1 trick you need to know to do it, wrap the object in parentheses.

```js
const someFunc= () => ({
  dogs: true,
  cats: false
})
```

```js
const mapStateToProps = state => ({
  profile: state.mentorReducer.mentorProfile
})
```

Useful if you like the no return syntax allowed by arrow functions that are one-liners

Reply:
Now I know that, thank you for sharing.